spec/features/basic_spec.rb

```diff
@@ -29,14 +29,14 @@
   end
 
   before do
-    rsolr_client = double('rsolr_client')
+    rsolr_client = instance_double(RSolr::Client)
```

Comment:
```suggestion
    rsolr_client = instance_double(RSolr::Client, send_and_receive: solr_facet_resp)
```

And then remove the next line.